scripts/db/2023.2.0/002_recover_vlans

```diff
@@ -0,0 +1,60 @@
+from collections import defaultdict
+from kytos.core.tag_ranges import range_difference
+
+def get_range(vlans, avoid) -> list[list[int]]:
+    """Convert available_vlans to available_tags.
+    From list[int] to list[list[int]]"""
+    result = []
+    if not vlans:
+        return result
+    vlans.sort()
+    i = 0
+    while i < len(vlans):
+        if vlans[i] in avoid:
+            i += 1
+        else:
+            break
+    if not vlans[i:]:
+        return result
+
+    start = end = vlans[i]
+    for tag in vlans[i+1:]:
+        if tag in avoid:
+            continue
+        if tag == end + 1:
+            end = tag
+        else:
+            result.append([start, end])
+            start = end = tag
+    result.append([start, end])
+    return result
+
+mef_eline = controller.napps[('kytos', 'mef_eline')]
+evcs = {evc_id: evc.as_dict() for evc_id, evc in mef_eline.circuits.items() if not evc.archived}
+
+in_use_tags = defaultdict(set)
+for evc_id, evc in evcs.items():
+    for link in evc["current_path"]:
+        svlan = link["metadata"]["s_vlan"]["value"]
+        intfa = link["endpoint_a"]["id"]
+        intfb = link["endpoint_b"]["id"]
+        in_use_tags[intfa].add(svlan)
+        in_use_tags[intfb].add(svlan)
+    for link in evc["failover_path"]:
+        svlan = link["metadata"]["s_vlan"]["value"]
+        intfa = link["endpoint_a"]["id"]
+        intfb = link["endpoint_b"]["id"]
+        in_use_tags[intfa].add(svlan)
+        in_use_tags[intfb].add(svlan)
+
```

Comment:
What about potentially missing vlans from EVPLs UNIs? Especially since version 2023.2.0 where it's allocating, can you review and double check this part? I can see a case where a UNI wasn't allocating before (the new script would consider it available when computing the tag range diff), then after a new update it can try to release then it'd would result in error at runtime later on.

Reply:
Added UNIs VLANs to the collection.